packages/deployment-service-ui/src/components/pages/releases.tsx

```diff
@@ -3,9 +3,10 @@ import { releaseServicePaginate, releaseVersionDeploy } from '@/platform-api/rel
 import { ReapitConnectSession, useReapitConnect } from '@reapit/connect-session'
 import { FlexContainerBasic, H3, Section } from '@reapit/elements-legacy'
 import { Button, Table } from '@reapit/elements-legacy'
-import { Loader, SnackProvider } from '@reapit/elements'
+import { Loader, StatusIndicator } from '@reapit/elements'
 import React, { useEffect, useState } from 'react'
 import { useParams } from 'react-router'
+// import useSnack from '@reapit/elements/src/hooks/use-snack'
```

Comment:
As above, should come from main manifest

Reply:
this one defo doesn't and I've checked. It's weird